apps/pie-docs/src/components/checkbox-group/overview/overview.md

```diff
@@ -0,0 +1,196 @@
+      src: "../../../assets/img/components/checkbox-group/modifiers-form-label-2.svg",
+      width: "258px",
+      alt: "Three horizontally aligned checkboxes without group label."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+### Assistive text
+
+Assistive text is used where needed to provide additional information or error / success messaging regarding the whole form element, and never to individual checkboxes. The assistive text is always used for an error state to provide the user with the information required to complete the form element.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/checkbox-group/modifiers-assistive-text.svg",
+    alt: "Three horizontally aligned checkboxes with group label and red assistive text underneath specifying error state.",
+    width: "258px"
+} %}
+
+---
+
+## Content
+
+All content should use sentence case.
+
+### Label
+
+- Always use clear and concise labels for checkbox group.
+
+---
+
+## Overflow
+
+### Multiple words overflow
+
+When the label exceeds the available width, the label will wrap onto a new line. Checkboxes with the same row remain top aligned for consistency, and any additional rows underneath move down to accommodate the additional content.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/checkbox-group/overflow.svg",
+    alt: "Six horizontally aligned checkboxes. First checkbox has a label that overflows to a second line. ",
+    width: "469px"
+} %}
+
+---
+
+## States
+
+Individual checkbox’s have their own interactive states: default, hover, active, and focus. However, some states apply to the group as a collective because they affect all checkboxes: error and disabled.
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox-group/states-error.svg",
+      width: "258px",
+      alt: "Three horizontally aligned checkboxes with group label and red assistive text underneath specifying error state."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/checkbox-group/states-disabled.svg",
+      width: "258px",
+      alt: "Three horizontally aligned checkboxes with group label. Checkbox boxes are greyed out to specify a disabled state."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Examples
+
+### LTR examples
+
+Here are some examples of the component in a left-to-right context:
+
+{% contentPageImage {
+    src:"../../../assets/img/components/checkbox-group/examples-ltr.svg",
+    alt: "Two vertically aligned checkboxes with a group label aligned to the right.",
+    width: "141px"
+} %}
+
+### RTL examples
+
+Here are some examples of the component in a right-to-left context:
+
+{% contentPageImage {
+    src:"../../../assets/img/components/checkbox-group/examples-rtl.svg",
+    alt: "Two vertically aligned checkboxes with a group label aligned to the left.",
```

Comment:
It looks like the checkboxes in this image have a different border-radius: 

<img width="179" alt="Screenshot 2024-07-29 at 15 31 28" src="https://github.com/user-attachments/assets/567c7cc4-82ba-48d6-9ac1-cb3c4e982e0f">

<img width="179" alt="Screenshot 2024-07-29 at 15 32 04" src="https://github.com/user-attachments/assets/0b1421d7-1e0c-4876-a9e7-c7154a11e1a0">

Reply:
Good catch, I'll reach out to designers to update 👍